maintainers/maintainer-list.nix

```diff
@@ -11996,6 +11996,16 @@
     githubId = 42300264;
     name = "wishfort36";
   };
+  witchof0x20 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.